sdk/identity/test-resources-post.ps1

```diff
@@ -67,7 +67,7 @@ Write-Host "Deploying Identity Docker image to ACR"
 az acr login -n $DeploymentOutputs['IDENTITY_ACR_NAME']
 $loginServer = az acr show -n $DeploymentOutputs['IDENTITY_ACR_NAME'] --query loginServer -o tsv
 $image = "$loginServer/identity-aks-test-image"
-docker build --no-cache -t $image "$workingFolder/AzureKubernetes"
+docker build --no-cache --build-arg REGISTRY="mcr.microsoft.com/mirror/docker/library/" -t $image "$workingFolder/AzureKubernetes"
```

Comment:
Alternatively, we could make this the default, but I need to check with @KarishmaGhiya  whether there's a reason for [this comment](https://github.com/Azure/azure-sdk-for-js/pull/30442/files#diff-3ebb2ded0468bcaea84005aafa1895287c50fe8fe6ce164bd01b286b5c2de84cR8) or if it was just copy-paste

Reply:
Very likely copy-paste from other Dockerfiles since that's what I did. :)
For Python, I just [defaulted](https://github.com/Azure/azure-sdk-for-python/commit/1f7dd2b37752d746668943e417709c5d99b17ecd) the registry in the Dockerfile to MCR, but this is fine, too.

Reply:
Fair enough! Thanks @pvaneck for reviewing 🙏